### PR TITLE
Fix bug and sync

### PR DESCRIPTION
**Pull request by vipwlb**

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C1%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703010753376403-205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/ApplicationCore/AzureBaseApplication/AzureBaseApplication.cs [hunk 1]
```diff
@@ -50,8 +50,19 @@ public abstract class AzureBasedApplicationBase
...
             }
             else if (!string.IsNullOrEmpty(aadClientSecret))
-            {
-                platformSettings = new ClientPlatformSettings(aadClientSecret, Guid.Parse(aadClientId));
+            {                
+                //TAP partners use below code path:
+                platformSettings = new ClientPlatformSettings
+                   (
+                    null,
+                     Guid.Parse(aadClientId),
```
Comment (jichemicrosoft) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/ApplicationCore/AzureBaseApplication/AzureBaseApplication.cs]: remove this and use the public developer flow

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/WebRole1/Web.config [hunk 2]
```diff
@@ -27,16 +27,10 @@
...
-    <add key="Microsoft.ServiceBus.ConnectionString" value="Endpoint=..." />
-    <add key="AppTokenCertThumbprint" value="ABCDEF1234567890ABCDEF1234567890ABCDEF12" />
-    <add key="AAD_ClientId" value="GUID" />
-    <add key="ApplicationEndpointId" value="sip:trustedEndpoint@tenantname.onmicrosoft.com" />
-    <add key="AudienceUri" value="https://base.url" />
-    <add key="CallbackUriFormat" value="https://base.url/callback?callbackContext={0}" />
-    <add key="ResourcesUriFormat" value="https://base.url/resources/{0}" />
+    <add key="AAD_ClientId" value="df2adc63-3db1-46ab-ad85-2a29f6440c63" />
+    <add key="AAD_ClientSecret" value="df2adc63-3db1-46ab-ad85-2a29f6440c63" />
+    <add key="ApplicationEndpointId" value="sip:appid@metio.onmicrosoft.com" />
```
Comment (jichemicrosoft) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/WebRole1/Web.config]: remove our settings , only put place holder here

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/ApplicationCore/AzureBaseApplication/AzureBaseApplication.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/ApplicationCore/AzureBaseApplication/AzureBaseApplication.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/WebRole1/Web.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/WebRole1/Web.config) | :white_check_mark:Succeeded | |
[Skype/WebSDK/docs/PTAuthAzureAD.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/WebSDK/docs/PTAuthAzureAD.md) | :white_check_mark:Succeeded | |
[Skype/WebSDK/docs/PTUIControlsConversationControl.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/WebSDK/docs/PTUIControlsConversationControl.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C13%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703130215001659-205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/ApplicationCore/AzureBaseApplication/AzureBaseApplication.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/ApplicationCore/AzureBaseApplication/AzureBaseApplication.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/WebRole1/Web.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/WebRole1/Web.config) | :white_check_mark:Succeeded | |
[Skype/WebSDK/docs/PTAuthAzureAD.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/WebSDK/docs/PTAuthAzureAD.md) | :white_check_mark:Succeeded | |
[Skype/WebSDK/docs/PTUIControlsConversationControl.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/WebSDK/docs/PTUIControlsConversationControl.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C13%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703130216072450-205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/docs/CORChatClient.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/CORChatClient.md) | :warning:Warning | | [Details](#user-content-Skype%2FTrusted-Application-API%2Fdocs%2FCORChatClient.md)
[Skype/TOC.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/TOC.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/docs/ApplicationCallFlow.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/ApplicationCallFlow.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/docs/BootstrapChatWidget.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/BootstrapChatWidget.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/docs/ChatCallflowDetail.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/ChatCallflowDetail.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/docs/CustomerChatCallflow.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/CustomerChatCallflow.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/docs/DiscoveryChatClient.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/DiscoveryChatClient.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/docs/ImplementingChatClientWithSkypeWebSDK.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/ImplementingChatClientWithSkypeWebSDK.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/docs/MessagingCallflow.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/MessagingCallflow.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/docs/PSTN.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/PSTN.md) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/ApplicationCore/AzureBaseApplication/AzureBaseApplication.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/ApplicationCore/AzureBaseApplication/AzureBaseApplication.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/WebRole1/Web.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/WebRole1/Web.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/Startup.Auth.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/Startup.Auth.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/WebApiConfig.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/WebApiConfig.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRSample.csproj](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRSample.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Controllers/CallbackController.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Controllers/CallbackController.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/prompt.wav](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/prompt.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptOne.wav](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptOne.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptTwo.wav](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptTwo.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Startup.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Startup.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Debug.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Debug.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Release.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Release.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/packages.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample.sln](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/AVBridgeJob.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/AVBridgeJob.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/App_Start/Startup.Auth.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/App_Start/Startup.Auth.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/App_Start/WebApiConfig.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/App_Start/WebApiConfig.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/AvBridgeSample.csproj](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/AvBridgeSample.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Common/DiagnosticLogger.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Common/DiagnosticLogger.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Common/LoggingContextActionFilter.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Common/LoggingContextActionFilter.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Common/SimpleEventChannel.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Common/SimpleEventChannel.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Common/UnityHelper.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Common/UnityHelper.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Controllers/CallbackController.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Controllers/CallbackController.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Global.asax](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Global.asax) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Global.asax.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Global.asax.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Properties/AssemblyInfo.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Resources/AVBridgeSample.wav](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Resources/AVBridgeSample.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Resources/CallCenterSample.wav](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Resources/CallCenterSample.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Startup.cs](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Startup.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Web.Debug.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Web.Debug.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Web.Release.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Web.Release.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Web.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/Web.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/packages.config](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeSample/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeToSipUriSample.sln](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AvBridgeToSipUriSample.sln) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
<a name="Skype%2FTrusted-Application-API%2Fdocs%2FCORChatClient.md"></a>
### [Skype/Trusted-Application-API/docs/CORChatClient.md](https://github.com/vipwlb/skype-docs/blob/master/Skype/Trusted-Application-API/docs/CORChatClient.md)
 - **Line 28**: **[Warning]** Invalid file link:(~/Trusted-Application-API/docs/CallFlow1.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C20%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201704200712143094-205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C21%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201704210046116563-205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request closed without merging**